controller.Makefile

```diff
@@ -131,7 +131,7 @@ lint-fix: golangci-lint ## Run golangci-lint linter and perform fixes
 
 .PHONY: build
 build: manifests generate fmt vet ## Build manager binary.
-	go build -o bin/manager cmd/main.go
```

Comment:
In Makefile there is a target that builds binary:
```
.PHONY: controller
controller:
	CGO_ENABLED=0 GOOS=${OS} GOARCH=${ARCH} go build -ldflags '${LDFLAGS}' -o ${OUTDIR}/${BINNAME} cmd/*.go
```

Do you think we can add these env vars from `controller` target to `build` target?